app-modules/meeting-center/tests/Tenant/Filament/Resources/BookingGroups/Pages/CreateBookingGroupTest.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use AdvisingApp\MeetingCenter\Filament\Resources\BookingGroups\Pages\CreateBookingGroup;
+use AdvisingApp\MeetingCenter\Models\BookingGroup;
+use AdvisingApp\MeetingCenter\Tests\Tenant\Filament\Resources\BookingGroups\Pages\RequestFactory\CreateBookingGroupRequestFactory;
+use AdvisingApp\Team\Models\Team;
+use App\Models\User;
+
+use function Pest\Laravel\actingAs;
+use function Pest\Laravel\assertDatabaseCount;
+use function Pest\Laravel\get;
+use function Pest\Livewire\livewire;
+use function Tests\asSuperAdmin;
+
+it('can render with proper permission', function () {
+    $user = User::factory()->create();
+
+    actingAs($user);
+
+    get(CreateBookingGroup::getUrl())
+        ->assertForbidden();
+
+    $user->givePermissionTo('group_appointment.view-any');
+    $user->givePermissionTo('group_appointment.create');
+
+    $user->refresh();
+
+    actingAs($user);
+
+    get(CreateBookingGroup::getUrl())
+        ->assertSuccessful();
+});
+
+it('validates the inputs', function (CreateBookingGroupRequestFactory $data, array $errors) {
+    asSuperAdmin();
+
+    $request = CreateBookingGroupRequestFactory::new($data)->create();
+
+    $user = User::factory()->create();
+
+    BookingGroup::factory()->for($user, 'createdBy')->create(['name' => 'Test Booking Group']);
```

Comment:
What is the purpose of creating this from the factory and asserting below? I would figure we shouldn't need to do this and would instead assert the DB is empty